src/builtins/core/duration/normalized.rs

```diff
@@ -289,7 +302,7 @@ impl NormalizedDurationRecord {
 #[derive(Debug)]
 struct NudgeRecord {
     normalized: NormalizedDurationRecord,
-    _total: Option<i128>, // TODO: adjust
+    _total: Option<FiniteF64>,
```

Comment:
nit: remove "_"

It was underscored because it was not be used. Now that it is in use, we don't need the underscore.